apps/web/components/getting-started/steps-views/SetupAvailability.tsx

```diff
@@ -22,15 +22,13 @@ const SetupAvailability = (props: ISetupAvailabilityProps) => {
   const { nextStep } = props;
 
   const router = useRouter();
-  let queryAvailability;
-  if (defaultScheduleId) {
-    queryAvailability = trpc.viewer.availability.schedule.get.useQuery(
-      { scheduleId: defaultScheduleId },
-      {
-        enabled: router.isReady,
-      }
-    );
-  }
+
+  const queryAvailability = trpc.viewer.availability.schedule.get.useQuery(
+    { scheduleId: defaultScheduleId as number | undefined },
```

Comment:
defaultScheduleId can still be null which can cause this error.

<img width="631" alt="Screenshot 2023-08-22 at 1 50 13 AM" src="https://github.com/calcom/cal.com/assets/53316345/84c37b25-5ee7-4320-b69b-3cbaab3fe56a">